sva/experiments/negatedackley2d/negatedackley2d.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from attrs import define, field
+
+from sva.monty.json import MSONable
+
+from ..base import ExperimentMixin, ExperimentProperties
+from ..campaign import CampaignBaseMixin
+
+
+@define
+class negatedAckley2d(ExperimentMixin, CampaignBaseMixin, MSONable):
```

Comment:
@tkshastry overall looks great! Only real criticism here is that class names should be [CamelCase](https://en.wikipedia.org/wiki/Camel_case) and start with a capital letter. Mind making those changes?

Reply:
Yup, changes made! Should be on the second commit.